### DISCUSSION TITLE
dist.ini

### DISCUSSION THREAD

**Discussion by jbeaujour** [sni/thruk#1120] [General]
Hello
The file dist.ini is not in 2.44-3 but in 2.40
How this file is upgrade : only on major version ?
We need for our reports the latest version of THRUK on GITHUB
Thanks

**Comment by sni** [comment]
Sounds like something doesn't work. All files with release information will be update from the `make version` which runs this script:
https://github.com/sni/Thruk/blob/master/script/thruk_version.sh#L73
So in theory, dist.ini should be updated. Although Thruk does not use that file itself. That's probably the reason why i didn't notice anything.

**Comment by sni** [comment]
should be fine now.
